pullrequest.rst

```diff
@@ -185,6 +185,11 @@ one or two discrepancies those can be fixed by the core developer who merges
 your pull request. But if you have systematic deviations from the style guides
 your pull request will be put on hold until you fix the formatting issues.
 
+.. note::
+   Pull requests with only code formatting changes are usually rejected. On the other
+   hand, fixing typos and grammar errors in documents and docstrings are very
```

Comment:
Grammar error ;-).  'fixing' is singular, so 'fixing ... is very welcome' is the correct grammar.  But 'is' after 4 subordinate plurals is a bit awkward.  I suggest deleting 'very' because at least one coredev, Guido, opposes spelling fixes.  But the average opinion of coredevs who have opined seems to be at least 'welcome'.
```suggestion
   hand, fixes for typos and grammar errors in documents and docstrings are
```